kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -5,8 +5,11 @@ map $http_accept $extension {
     application/json json;
 }
 
-{% if nginx_storage_ssl %}
-server {
+{% if nginx_storage_allow_http %}
+ server {
+    listen 80       default_server;
+{% else %}
```

Comment:
Same as above. This does not enable HTTPS.

Reply:
Done.